### PR TITLE
Boyscouting

### PR DESCRIPTION
**Pull request by amolenaar**
Some cleanup, all over the place.

### PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [X] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes

### CODE REVIEW

File: gaphor/application.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def load_services(self, services=None):
 
         Service should provide an interface gaphor.abc.Service.
         """
-        for ep in pkg_resources.iter_entry_points("gaphor.services"):
+        for ep in importlib_metadata.entry_points()["gaphor.services"]:
```
Comment (danyeaw) [inline gaphor/application.py]: Nice, I didn't know about that package :+1:

**Comment by danyeaw** [comment]
:1st_place_medal:

File: gaphor/application.py (hunk 1 above)
Comment (amolenaar) [reply gaphor/application.py]: I ran into it by accident. It looks like it’s used a lot already. It was part of our transient dependencies.

**Comment by amolenaar** [comment]
I need to change one more thing: delete the inject() function.

**Pull request merged**